Cargo.toml

```
@@ -13,3 +13,5 @@ ariel-os-boards = { path = "build/imports/ariel-os/src/ariel-os-boards" }
 [dev-dependencies]
 embedded-test = { version = "0.6.1", features = ["ariel-os"] }
 {% endif -%}
+[lints.rust]
+unexpected_cfgs = "allow"
```

Could you instead use `cfg(context, values(any()))` to allow the `context` `cfg` specifically?

```suggestion

[lints.rust]
unexpected_cfgs = "allow"
```